Gopkg.lock

```diff
@@ -106,11 +104,10 @@
     "rpc",
     "trie",
     "whisper/mailserver",
-    "whisper/notifications",
     "whisper/whisperv5"
   ]
-  revision = "5b2cc44bf2b32bb482def02d7c8fa32ba08d0bf4"
-  source = "https://github.com/status-im/go-ethereum.git"
+  revision = "4bb3c89d44e372e6a9ab85a8be0c9345265c763a"
```

Comment:
Can you elaborate?
We won't be able to keep the same revision SHA in this PR because we point to another repository.
In this diff I don't see anything changed in `go-ethereum`, just one main file + a few test utils.

Reply:
Nevermind, I'm still partly asleep :-) I was assuming that the vendor folder would contain the unpatched version, which of course doesn't make sense.